.gitignore

```diff
@@ -1,3 +1,5 @@
+diagnostic.data/
```

Comment:
Just curious -- What tool creates this directory that we need to ignore?

Reply:
not sure